.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -4,24 +4,52 @@ about: Request our thoughts on your suggestion for a new feature for Homebrew.
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+<details>
```

Comment:
This will hide everything by default, no? That seems undesirable.

Reply:
This hides the contributor guidelines from the rendered comment only.
Contributor see them when they're working on the contribution. 

My idea was to help maintainers concentrate on the contribution rather than parsing though the static wording of the template.

Reply:
I see. Well, I'm open to trying it out and reverting (this particular bit) if it makes life harder.